db/migrate/20180626125655_rename_user_maintenance_zone_record.rb

```diff
@@ -0,0 +1,31 @@
+class RenameUserMaintenanceZoneRecord < ActiveRecord::Migration[5.0]
+  class Zone < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+
+    validates_presence_of   :name
+
+    MAINTENANCE_ZONE_NAME = "__maintenance__".freeze
+  end
+
+  def up
+    say_with_time("Renaming user-defined Maintenance Zone") do
+      zone = Zone.in_my_region.where(:name => Zone::MAINTENANCE_ZONE_NAME).first
```

Comment:
If the name is unique can't you just do `.find_by()` instead of `.where().first`?